docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
```

Comment:
@gabriel-lima only as a suggestion, what do you think to use docker-compose version 3? Is the newest version and we eliminate the need to update this in the future.